apps/docs/src/app/core/component-docs/table/table-docs.component.html

```diff
@@ -164,6 +164,21 @@
 
 <separator></separator>
 
+<fd-docs-section-title [id]="'with-row-unnavigatable'" [componentName]="'table'">
+    Table with Non-navigatable Row
+</fd-docs-section-title>
+<description>
+    A row can be non-navigatable by setting the property <code>[navigatable]="false"</code>, for both regular and Pop In modes.
+    Non-navigatable rows will disable interaction and navigation. <br>
+    To hide the arrow icon, set <code>[glyph]="''"</code> and <code>[navigation]="false"</code> for the <code>fd-table-icon</code> directive.
```

Comment:
I wouldn't use empty string for determining the disabled icon, maybe boolean, or empty as a default would be better. Or even getter, that checks the conditions for setting the icon

Reply:
Thanks I'll leave it empty instead of using ```''```